components/raftstore-v2/src/raft/storage.rs

```diff
@@ -61,6 +64,15 @@ pub struct Storage<ER> {
     logger: Logger,
 }
 
+impl<ER: RaftEngine> Deref for Storage<ER> {
```

Comment:
It's not implemented on purpose to avoid confusion.

Reply:
Fine.